apix-impl/src/integration-test/java/eu/xenit/apix/tests/search/SearchServiceTest.java

```diff
@@ -322,4 +321,31 @@ private void withLocale(Locale locale, AuthenticationUtil.RunAsWork<Object> work
      */
     abstract protected void withTestFacets(AuthenticationUtil.RunAsWork<Object> work) throws Exception;
 
+
+    @Test
+    public void testNotFilteredOut() {
+        SearchSyntaxNode node =
+                new QueryBuilder()
+                        .startOr()
+                        .property("{http://www.alfresco.org/model/system/1.0}node-dbid", "9223372036854775807")
+                        .end()
+                        .create();
+        SearchQuery searchQuery = new SearchQuery();
+        searchQuery.setQuery(node);
+        searchService.query(searchQuery);
+    }
+
+    @Test
+    public void testNothingLeftAfterFiltering() {
```

Comment:
this is EXACTLY the same test as above ?

Reply:
It might seem so, but the long value is 1 greater, crossing the max value boundary

Reply:
Ok, need to make that more obvious somehow ;-)

Reply:
I will add it to the method name, together with the `throws exception`